openpype/modules/kitsu/plugins/publish/integrate_kitsu_note.py

```diff
@@ -62,13 +65,41 @@ def process(self, context):
 
             # Check if any status condition is not met
             allow_status_change = True
-            for status_cond in self.status_conditions:
+            for status_cond in self.status_change_conditions[
+                "status_conditions"
+            ]:
                 condition = status_cond["condition"] == "equal"
                 match = status_cond["short_name"].upper() == shortname
                 if match and not condition or condition and not match:
                     allow_status_change = False
                     break
 
+            if allow_status_change:
+                # Get families
+                families = {
+                    instance.data.get("family")
+                    for instance in context
+                    if instance.data.get("publish")
+                }
+
+                allow_status_change = False
```

Comment:
This is weird that you don't use the same logic as for statuses, because doing this introduces the same default use case issue: if there is no any item in`family_requirements`, `allow_status_change` will remain `False` and the note status won't be changed, while we expect it to be if no any family is listed. I think you must stick to your design above.